backend/python/app/rest/entity_routes.py

```diff
@@ -1,84 +1,83 @@
-from flask import Blueprint, current_app, request

```

Comment:
nit: Would you be down to try reverting entity_routes.py, models/user.py, and models/entity.py? Do you know what might have caused this linting issue?